CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog for `wai-saml2`
 
+-   Added `validation` test ([#52](https://github.com/mbg/wai-saml2/pull/52) by [@fumieval](https://github.com/fumieval))
```

Comment:
This change probably doesn't need to be in the changelog, but I don't feel strongly either way. So happy for you to leave it in if you want.